tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1568,6 +1569,44 @@ public void Index_Example_2()
             result.Should().Be("cuisine_1_name_1");
         }
 
+        [Fact]
+        public void Exploiting_The_Power_Of_Arrays_Example()
+        {
+            RequireServer.Check().Supports(Feature.ArrayFilters);
+
+            var document = new BsonDocument
+            {
+                { "_id", 1 },
+                { "a", new BsonArray { new BsonDocument("b", 0), new BsonDocument("b", 1) } }
+            };
+
+            var arrayUpdatesTestCollectionName = "arrayUpdatesTest";
+            var testDatabase = client.GetDatabase("test");
+            testDatabase.DropCollection(arrayUpdatesTestCollectionName);
+            var arrayUpdatesTestCollection = testDatabase.GetCollection<BsonDocument>(arrayUpdatesTestCollectionName);
+            arrayUpdatesTestCollection.InsertOne(document);
+
+            // Start Exploiting the power of arrays example
```

Comment:
Make sure this delimiter matches what the doc team is looking for.

Reply:
I asked in the tickets. So as soon as I will have the answer, I will update this